lib/main.coffee

```diff
@@ -1,6 +1,5 @@
-{CompositeDisposable} = require 'atom'
```

Comment:
Usually we don't defer this require since it is always precached, https://github.com/atom/atom/blob/14a7caa46a4307174dbf855a3f3cdeeed853f5b8/src/package-manager.coffee#L326-L328

Did you notice a difference if it was required on load vs. activate?